quicfire_tools/inputs.py

```diff
@@ -722,3 +745,356 @@ def from_file(cls, directory: str | Path):
             maximum_firebrand_ignitions=maximum_firebrand_ignitions,
             minimum_landing_angle=minimum_landing_angle,
             maximum_firebrand_thickness=maximum_firebrand_thickness)
+
+
+class QUIC_fire(InputFile):
+    """
+    Class representing the QUIC_fire.inp input file. This file
+    contains the parameters relating to the fire simulation and
+    outputs.
+
+    Parameters
+    ----------
+    nz : int
+        Number of fire grid cells in the z-direction.
+    output_times: int | OutputTimes
```

Comment:
Maybe this is what the tag is for. Can we make these individual attributes instead of an int or OutputTimes class?

Reply:
Yep see comment above